### DISCUSSION TITLE
OLED Display SSD1327

### DISCUSSION THREAD

**Discussion by ghost** [tbnobody/OpenDTU#2243] [Ideas]
Hallo Zusammen,

wäre es möglich ein weiteres Display mit in die möglichen Displays zu nehmen?
Ich fände das OLED Display SSD1327 nicht schlecht.

Danke :-)

**Comment by ghost** [comment]
Hi,
ich bin kein Entwickler, allerdings ist mir aufgefallen, dass es OLEDs mit SSD1327 Treiber für I2C- sowie SPI-Interface , die in verschiedenen Auflösungen und von verschiedenen Herstellern gibt.
Ich glaube nicht, das alle unterstützt würden.
Schlage mal eins vor.

Du könntest auch einen Zweig von diesem Repository machen und inclusive der Web App selbst kompilieren; das müsstest du allerdings bei jedem Update aktualisieren und jedes mal auch wiederholen.

**Reply by ghost** [reply]
Also da würde ich dann mal das Waveshare Modul ins rennen werfen.
Hat eine Auflösung von 128x128px und kann SPI / I2C. Das ist ein 1,5" OLED
Display. Natürlich in der RGB Variante.

Das mit dem Zweig habe ich zwar ansatzweise verstanden, also das was Du 
vorgeschlagen hast, meine progammierkenntnisse reichen aber leider nicht 
aus, um das zu machen. Ich kann es schlichtweg nicht.

**Comment by Tpaydt** [comment]
Was erhoffst du von dem 1.5" Display?
Es ist kaum größer zu dem 1.3 was schon unterstützt wird. Falls du es besser ablesen willst nimm das 2.4 SSD1309, da sieht man wenigstens was.

**Reply by ghost** [reply]
Das einzige was ich mir erhoffe ist, dass ich das nutzen kann was ich noch zu Hause habe und nichts neu kaufen muss.

**Comment by ghost** [comment]
Dein Display zu integrieren wäre für mich nicht das ganz große Problem. Das sind, wenn man ein anderes heraus schmeißt, eine  / zwei  Zeilen Code.
Aber:
128x128 ist eine noch nicht vorhandene Auflösung. Mangels ähnlichem Display kann ich daher nicht sagen, ob es zu Darstellungsfehlern kommt. Es könnte sein dass nur die Hälfte des Bildschirmes genutzt wird., oder alles versetzt angezeigt wird.
Inwiefern die Displayauflösung im OpenDTU Programcode eine Rolle spielt, habe ich keine Ahnung, es ist ziemlich viel Code. Ich bin selber Neuling und tue mich damit selber schwer.
Bei jedem Update müsste jedes Mal der eigene Zweig aktualisiert werden, das klappt auch nie auf Anhieb, zumindest nicht bei mir.
Weiterhin kann ich auch nicht garantieren, das ich immer Zeit habe das zu machen und eine Firmware zur Verfügung stelle. Das kann schon mal ein halbes Jahr oder länger dauern.

Beim ALI-China-Mann kosten Displays teilweise weniger wie das Porto, Vielleicht könntest du dich doch zu einem neuen durchringen.
Vielleicht wirst du aber auch erhört.
